test/e2e/e2e.go

```diff
@@ -1871,3 +1872,223 @@ func getNodePodCIDR(nodeName string) (string, error) {
+			framework.RunKubectl(ovnNs, "exec", db_pod_name, containerFlag, "--", "rm", dirDB+DBFileNameToDelete)
+			//check db files- make sure its deleted
+			out, _ := framework.RunKubectl(ovnNs, "exec", db_pod_name, containerFlag, "--", "ls", dirDB)
+			out = strings.TrimSuffix(strings.ReplaceAll(out, "\n", ","), ",")
+			if out == "" {
+				framework.Logf("all db files deleted")
+			} else {
+				framework.Logf("cur db files:%s", out)
+			}
+			if strings.Contains(out, DBFileNameToDelete) {
+				errStr = "db not deleted:" + DBFileNameToDelete
+				framework.Failf(errStr)
+			}
+		}
+
+		//delete the db-pod in order to emulate the cluster/pod restart
+		framework.Logf("deleting db pod:%s", db_pod_name)
+		deletePod(f, ovnNs, db_pod_name)
+
+		//wait (let the new db pod to be ready)
+		framework.Logf("wait %v", waitingPeriod)
+		time.Sleep(waitingPeriod)
+
+		//check db files existence
+		checkDbFilesExistence(db_pod_num)
+	}
+	singlePodConnectivityTest := func(f *framework.Framework, podName string) {
+		framework.Logf("Running container which tries to connect to 8.8.8.8 in a loop")
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+
+		go checkContinuousConnectivity(f, "", podName, "8.8.8.8", 53, 30, podChan, errChan)
+		testPod := <-podChan
+
+		framework.Logf("Test pod running on %q", testPod.Spec.NodeName)
+		framework.ExpectNoError(<-errChan)
+	}
+	twoPodsConnectivityTest := func(f *framework.Framework,
+		node1Name string, node2Name string,
+		syncChan chan string, errChan chan error) {
+		const (
+			pod1Name                  string        = "connectivity-test-pod1"
+			pod2Name                  string        = "connectivity-test-pod2"
+			port                      string        = "8080"
+			timeIntervalBetweenChecks time.Duration = 2 * time.Second
+		)
+
+		var (
+			command = []string{"/agnhost", "netexec", fmt.Sprintf("--http-port=" + port)}
+		)
+		createGenericPod(f, pod1Name, node1Name, f.Namespace.Name, command)
+		createGenericPod(f, pod2Name, node2Name, f.Namespace.Name, command)
+
+		// wait for pods setup to return a valid address
+		wait.PollImmediate(retryInterval, retryTimeout, func() (bool, error) {
+			for _, podName := range []string{pod1Name, pod2Name} {
+				kubectlOut := getPodAddress(podName, f.Namespace.Name)
+				srcIP := net.ParseIP(kubectlOut)
+				if srcIP == nil {
+					return false, nil
+				}
+			}
+			return true, nil
+		})
+		pod2IP := getPodAddress(pod2Name, f.Namespace.Name)
+
+		ginkgo.By("Checking initial connectivity from one pod to the other and verifying that the connection is achieved")
+
+		stdout, err := framework.RunKubectl(f.Namespace.Name, "exec", pod1Name, "--", "curl", fmt.Sprintf("%s/hostname", net.JoinHostPort(pod2IP, port)))
+
+		if err != nil || stdout != pod2Name {
+			framework.Failf("Error: attempted connection to pod %s found err:  %v", pod2Name, err)
+		}
+
+		syncChan <- "connectivity test pods are ready"
+
+	L:
+		for {
```

Comment:
There is a function, `checkContinuousConnectivity` in e2e.go. Can you use that?

Reply:
(for the record - I used `checkContinuousConnectivity` on the connectivity test with the single pod.)
I believe that on this case the function is not suitable.

```go
func checkContinuousConnectivity(f *framework.Framework, nodeName, podName, host string, port, timeout int, podChan chan *v1.Pod, errChan chan error) {
	contName := fmt.Sprintf("%s-container", podName)

	command := []string{
		"bash", "-c",
		"set -xe; for i in {1..10}; do nc -vz -w " + strconv.Itoa(timeout) + " " + host + " " + strconv.Itoa(port) + "; sleep 2; done",
	}
```
you can see that the `command` for the new pod is a for loop that run about 20-25 seconds (=constant time).I want to make sure that the connectivity test between the two pods will run as long as the disruption is taking place so using `checkContinuousConnectivity` will not fit here.